lib/Extractor/Candidates.cpp

```diff
@@ -386,6 +386,29 @@ Inst *ExprBuilder::build(Value *V) {
       }
       return IC.getPhi(BI.B, Incomings);
     }
+  } else if (auto EV = dyn_cast<ExtractValueInst>(V)) {
+    Inst *L = get(EV->getOperand(0));
+    ArrayRef<unsigned> Idx = EV->getIndices();
+    // NOTE : extractvalue instruction can take set of indices. Most of the
+    // times we just pass one index value, i.e. why I am extracting 0th index
+    // always, Idx[0]. If required, we can fix it by iterating over set of
+    // indices. Additonally, we harvest extractvalue instruction only if the
+    // extracted value comes from an overflow instruction. Otherwise, we simply
```

Comment:
What happens if you harvest an ``extractvalue`` instruction that has multiple indices? Does it change the semantics of the instruction when you extract only the index 0?